app/src/main/java/com/fcaputo/parkingapp/mvp/model/ParkingSettings.kt

```diff
@@ -0,0 +1,7 @@
+package com.fcaputo.parkingapp.mvp.model
+
+import com.fcaputo.parkingapp.utils.Constants
+
+object ParkingSettings {
+    var size: Int = Constants.SIZE_NOT_SET_INT
```

Comment:
you can rename this to ZERO_INT for example to be more general and use it in other places

Reply:
Done